openedx/adg/common/user_authn_override/forms.py

```diff
@@ -0,0 +1,135 @@
+"""
```

Comment:
Do we need this now?

Reply:
Yes we need this. It works like we use basic fields that we can use from userprofile and the DEFAULT_FIELDS will be added as a MUST and then at the end our custom form fields will be appended.